src/posts/paper/14-research-sources.ts

```diff
@@ -377,7 +377,39 @@ Control Group: \`351 / 19544 * 1 + 54 / 19162 * 0.42 = 0.019\`
 Vaccinated Group: \`117 / 19514 * 1 + 22 / 19301 * 0.42 = 0.0065\`
 Contagiousness ratio: \`0.0062 / 0.019 = 0.34\`
 
-From this we estimate that the microCOVID multiplier for the Johnson and Johnson vaccine is 0.38 14 days after injection, dropping down to 0.33 29 days post infection. Because these two multipliers are close together and the higher multiplier only covers weeks 2-4, for simplicity’s sake we recommend using a multiplier of 1/3x for 14 days post-vaccination and onwards. 
+From this we estimate that the microCOVID multiplier for the Johnson and Johnson vaccine is 0.38 14 days after injection, dropping down to 0.33 29 days post infection. Because these two multipliers are close together and the higher multiplier only covers weeks 2-4, for simplicity’s sake we recommend using a multiplier of 1/3x for 14 days post-vaccination and onwards.
+
+### Sputnik V (Gamelaya Research)
+
+The efficacy of the Sputnik V vaccine produced by the Gamelaya Research Institute is based on their phase 3 trial information [published in Lancet](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(21)00234-8/fulltext). Our calculation is similar to the one used for Moderna & Pfizer. Non-symptomatic cases were not measured in this study, so based on Byambasuren et al. and measurements for other vaccines, we estimate non-symptomatic cases to be 17% of all cases in the control group and 50% in the vaccinated one. As the control and the vaccinated group had a different size, we also have to normalize the adjusted infectiousness figures.
```

Comment:
Nit: please note that the 50% is extrapolating from the CDC's Moderna/Pfizer study.